tests/test_dislocation.py

```diff
@@ -55,7 +57,8 @@
 class TestDislocation(matscipytest.MatSciPyTestCase):
     """Class to store test for dislocation.py module."""
 
-    @unittest.skipIf("atomman" not in sys.modules, 'requires Stroh solution from atomman to run')
+    @unittest.skipIf("atomman" not in sys.modules,
```

Comment:
I recall a plan to replace atomman with our own Stroh solution. Is that still something you'd like to do? Doesnt't have to be be as part of this PR

Reply:
Yes I would like to do that because atomman has too many dependencies. Same goes for the Vitek plots and this is work in progress.